util/common/src/main/java/org/hibernate/search/util/common/reflect/impl/MethodHandleValueReadHandle.java

```diff
@@ -44,7 +45,7 @@ public T get(Object thiz) {
 			if ( e instanceof InterruptedException ) {
 				Thread.currentThread().interrupt();
 			}
-			throw log.errorInvokingMember( member, thiz, e );
+			throw log.errorInvokingMember( member, Throwables.safeToString( e, thiz ), e );
```

Comment:
Why haven't we change the catch into:
~~~ java
catch (RuntimeException | IllegalAccessException e) {
  throw log.errorInvokingMember( field, Throwables.safeToString( e, thiz ), e );
}
~~~
as we did for `FieldValueReadHandle` and `MethodValueReadHandle`?

Reply:
Because `MethodHandle.invoke` doesn't throw an `IllegalAccessException`. From what I can tell, that exception is thrown when creating the handle, if ever, but not upon execution.